### PR TITLE
#882 create a shared project

### PR DESCRIPTION
**Pull request by nashtech-huyphamphu**

### CODE REVIEW

File: cart/pom.xml [hunk 1]
```diff
@@ -48,6 +49,11 @@
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.yas</groupId>
+            <artifactId>common-library</artifactId>
+            <version>1.0-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (nashtech-huyphamphu) [reply cart/pom.xml]: I have updated the code as comment.

File: common-library/src/main/java/com/yas/commonlibrary/CommonLibraryApplication.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import com.yas.commonlibrary.config.ServiceUrlConfig;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+
+@SpringBootApplication
+@EnableConfigurationProperties(ServiceUrlConfig.class)
+public class CommonLibraryApplication {
+
```
[the remark this answers is not in the document]
Comment (nashtech-huyphamphu) [reply common-library/src/main/java/com/yas/commonlibrary/CommonLibraryApplication.java]: I have updated the code as comment.

File: common-library/src/main/java/com/yas/commonlibrary/config/ServiceUrlConfig.java
[the remark this answers is not in the document]
Comment (nashtech-huyphamphu) [reply common-library/src/main/java/com/yas/commonlibrary/config/ServiceUrlConfig.java:1]: I intended to do this, but I will implement it in the next pull request.

File: cart/src/main/java/com/yas/cart/exception/ApiExceptionHandler.java
[the remark this answers is not in the document]
Comment (nashtech-huyphamphu) [reply cart/src/main/java/com/yas/cart/exception/ApiExceptionHandler.java:1]: I intended to do this, but I will implement it in the next pull request.

**Review by VanCongBang — APPROVED** [review]

**Review by khanhtrand — APPROVED** [review]

**Pull request merged**